core-web/src/main/java/org/silverpeas/core/webapi/node/NodeType.java

```diff
@@ -47,7 +47,9 @@
   @XmlEnumValue("bin")
   BIN("bin"),
   @XmlEnumValue("tovalidate")
-  TO_VALIDATE("tovalidate");
+  TO_VALIDATE("tovalidate"),
+  @XmlEnumValue("nonvisiblepubs")
+  NON_VISIBLE_PUBS("nonvisiblepubs");
```

Comment:
NON_VISIBLE_PUBS is a non generic name as it refers explicitly the publications as being non visible. The name should be more generic like for example NON_VISIBLE or NON_VISIBLE_CONTRIB: the node is about non visible contributions (whatever they are: a publication, a media, ...) as the node is a generic and transversal object used to categorize others contributions in a tree.